dbschema/src/main/java/org/folio/rest/persist/ddlgen/Index.java

```diff
@@ -76,41 +77,44 @@ public void setSqlExpression(String sqlExpression) {
   public String getFinalSqlExpression(String tableLoc) {
     if (this.getMultiFieldNames() == null && this.getSqlExpression() == null) {
       return this.fieldPath;
-    } else if ( this.getSqlExpression() != null) {
+    }
+    if ( this.getSqlExpression() != null) {
       return this.getSqlExpression();
-    } else {
-      String [] splitIndex = this.getMultiFieldNames().split(" *, *");
+    }
 
-      StringBuilder result = new StringBuilder("concat_space_sql(");
-      for (int i = 0;i < splitIndex.length;i++) {
-        if(i != 0) {
-          result .append(" , ");
-        }
-        appendExpandedTerm(tableLoc, splitIndex[i],result);
+    String [] splitIndex = this.getMultiFieldNames().split(" *, *");
+
+    StringBuilder result = new StringBuilder("concat_space_sql(");
+    for (int i = 0; i < splitIndex.length; i++) {
+      if(i != 0) {
```

Comment:
I do like a space between if and the (.. Just me being pedantic

Reply:
Fixed all code formatting issues.